internal/operator-controller/rukpak/bundle/config.go

```diff
@@ -0,0 +1,108 @@
+package bundle
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/apimachinery/pkg/util/validation"
+	"sigs.k8s.io/yaml"
+
+	"github.com/operator-framework/api/pkg/operators/v1alpha1"
+)
+
+type Config struct {
+	WatchNamespace string `json:"watchNamespace"`
+}
+
+// UnmarshallConfig returns a deserialized and validated *bundle.Config based on bytes and validated
+// against rv1 and the desired install namespaces. It will error if:
+// - rv is nil
+// - bytes is not a valid YAML/JSON object
+// - bytes is a valid YAML/JSON object but does not follow the registry+v1 schema
+// if bytes is nil a nil bundle.Config is returned
+func UnmarshallConfig(bytes []byte, rv1 *RegistryV1, installNamespace string) (*Config, error) {
+	if bytes == nil {
+		return nil, nil
+	}
+	if rv1 == nil {
```

Comment:
if `nil` is not allowed, why do we not just `RegistryV1` instead a pointer to it?